app/js/app.js

```diff
@@ -133,4 +133,14 @@ alertaApp.config(['config', '$authProvider',
       clientId: config.client_id,
       authorizationEndpoint: config.keycloak_url+'/auth/realms/'+config.keycloak_realm+'/protocol/openid-connect/auth'
     });
+    $authProvider.oauth2({
+      name: 'pingfederate',
+      url: config.endpoint+'/auth/pingfederate',
+      redirectUri: config.redirectUri,
```

Comment:
Make this `redirectUri: window.location.origin,`

Reply:
this was my original implementation but because `/` was not appended in the url I had to add a parameter

Reply:
Why not `redirectUri: window.location.origin + '/'` then?

Reply:
why didnt I think of it! I have tested the suggested change & committed it.